server/models/TrustModel.js

```diff
@@ -0,0 +1,52 @@
+const expect = require('expect-runtime');
+const knex = require('../../server/database/knex');
+const HttpError = require("./HttpError");
+const EntityModel = require("./EntityModel");
+
+class TrustModel{
```

Comment:
I think writing model with class would bring better flexibility and testable. Comparing to just a static object.